app/javascript/app/pages/settings/settings-component.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { PureComponent } from 'react';
+import Header from 'components/header';
+import { PropTypes } from 'prop-types';
+import SettingsItem from './settings-item';
+import styles from './settings-styles.scss';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Settings extends PureComponent {
+  render() {
+    const { user } = this.props;
+    return (
+      <div className={styles.page}>
+        <Header title="Settings" className={styles.header} />
+        <div className={styles.wrapper}>
+          <SettingsItem
+            title={'Personal info'}
+            editActionLink={'/settings/info'}
+          >
+            {user && (
```

Comment:
wdyt about moving it to another component like `UserItem`?

Reply:
I think this is too little for a component. I extracted it into a renderUser function